README.md

```diff
@@ -128,10 +128,16 @@ A special `dev` command is available to modify an existing running server.
 Examine the [`docker`](docker) script for available commands.
 For example:
 
-Start the server with `NOCLEAN=true`:
+Start a particular server with `NOCLEAN=true`:
 ```
 NOCLEAN=true .omero/docker cli
 ```
+
+or start an independent server:
```

Comment:
Do you think it's worth keeping both options, or just have `.omero/docker dev start_up` as the recommended way?

Reply:
I think there are a couple of simplifications we can add. `NOCLEAN` was probably never a particularly good design from my side. I'd suggest we get this in but keep hacking on the usability of the repository.